chart/templates/deployment.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: poc-acapy-multitenant
+  labels:
+    app: poc-acapy-multitenant
+    chart: poc-acapy-multitenant-0.1.0
+    release: poc-acapy-multitenant
+    heritage: Helm
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: poc-acapy-multitenant
+      release: poc-acapy-multitenant
+  template:
+    metadata:
+      annotations:
+        # needs to be updated I suppose
+        iam.amazonaws.com/role: arn:aws:iam::715299534867:role/gid-dev-poc-acapy-multitenant
+      labels:
+        app: poc-acapy-multitenant
+        release: poc-acapy-multitenant
+    spec:
+      containers:
+        - name: poc-acapy-multitenant
+          # needs to be updated with published image value
+          # image: "715299534867.dkr.ecr.us-east-1.amazonaws.com/poc-acapy-multitenant:2a333a454c04f014aa02895249b2a98d4d924b78" 
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: mainport
+              containerPort: 9836
+          resources:
+            limits:
+              cpu: 500m
+              memory: 768Mi
+            requests:
+              cpu: 250m
+              memory: 192Mi
+          # should readinessProbe and startupProbe be present? 
+          readinessProbe:
+            httpGet:
+              path: poc-acapy-multitenant/health/ready
```

Comment:
Are you sure the probes are accessible on this address?

Reply:
nope. I left a comment should it be there at all? :) if so we should do smth with it